texastribune/queso-ui
# New footer docs and mobile footer redesign

#### What's this PR do?
Adds a new variation of the site footer to the docs with two columns on mobile instead of three

##### Classes added (if any)
- .c-site-footer__inner--standard
- .c-site-footer__links--standard

#### Why are we doing this? How does it help us?

To make our footer more mobile-friendly ([according to Google](https://web.dev/tap-targets/) 🤷‍♀️ )

The mobile layout I had to add to fix our lighthouse score is a little complicated. That got me thinking that we may just want to build it in as a variation to prevent having to copy/paste it everywhere the standard footer is used, which is a growing number of repos.


#### How should this be manually tested?
`npm run dev`

See: [The "standard" site footer](localhost:8080/modifier/c-site-footer__inner--standard/)


#### Does this introduce a breaking change where queso-ui is used in the wild? If so, is there a relevant branch/PR to accompany this release?

The donations account page footer will get a few more styles than it actually needs, but that seems fine to me. Everywhere else, we'll be able to delete some of the repo-specific CSS.


The point of this is to fix our small tap targets flagged in lighthouse, so for that reason I'm thinking that's considered a patch
